app/services/doc_auth/socure/responses/docv_result_response.rb

```diff
@@ -105,6 +112,39 @@ def read_pii
           )
         end
 
+        def log_verification_request
+          user_id = nil
+          if !document_capture_session&.user.nil?
+            user_id = document_capture_session.user.uuid
+          end
+          @analytics.idv_socure_verification_data_requested(
+            success: doc_auth_success?,
+            errors: nil,
+            exception: nil,
+            async: false,
```

Comment:
we could _default_ to true, but to make this dynamic we'd either pass it in the constructor or see if there is a way to know from ActiveJob (perform_now vs perform_later)